src/VisualStudio/Core/Def/Implementation/ExtractInterface/ExtractInterfaceDialog.xaml.cs

```diff
@@ -27,13 +27,21 @@ internal partial class ExtractInterfaceDialog : DialogWindow
         public string ExtractInterfaceDialogTitle { get { return ServicesVSResources.Extract_Interface; } }
         public string NewInterfaceName { get { return ServicesVSResources.New_interface_name_colon; } }
         public string GeneratedName { get { return ServicesVSResources.Generated_name_colon; } }
-        public string NewFileName { get { return ServicesVSResources.New_file_name_colon; } }
+        public string SelectDestinationFile { get { return ServicesVSResources.Location_colon; } }
+        public string SelectCurrentFileAsDestination { get { return ServicesVSResources.Add_to_existing_file; } }
+        public string SelectNewFileAsDestination { get { return ServicesVSResources.New_file_name_colon; } }
         public string SelectPublicMembersToFormInterface { get { return ServicesVSResources.Select_public_members_to_form_interface; } }
         public string SelectAll { get { return ServicesVSResources.Select_All; } }
         public string DeselectAll { get { return ServicesVSResources.Deselect_All; } }
         public string OK { get { return ServicesVSResources.OK; } }
         public string Cancel { get { return ServicesVSResources.Cancel; } }
 
+        private const string _currentFileDestinationTag = "CurrentFileDestinationTag";
+        private const string _newFileDestinationTag = "NewFileDestinationTag";
+
+        public string CurrentFileDestinationTag { get { return _currentFileDestinationTag; } }
+        public string NewFileDestinationTag { get { return _newFileDestinationTag; } }
```

Comment:
what are these for?

Reply:
When I thought binding to the tag was the right way to go, and forgot to remove :)